src/seedu/addressbook/AddressBook.java

```diff
@@ -199,7 +199,7 @@
      * ====================================================================
      */
     public static void main(String[] args) {
```

Comment:
bad method names and making methods do more than what they are meant for does make it smelly! Several other things also do. for e.g., bad variable names, lack of comments, bad comments, lack of SLAP.